content/en/account_management/audit_logs.md

```diff
@@ -0,0 +1,107 @@
+
+## Configuration
+
+### Event types
+
+Audit Logs is configurable by event type. Enabled events are specific to actions that take place in the Datadog application. The following event types are supported for Audit Logs:
+
+- Access Management: Manage access, such as roles and users
+- Authentication: Successful and failed login attempts
+- Dashboard: Additions, deletions, and changes to dashboard resources
+- Log Management: Additions, deletions, and changes to log management resources
+- Monitor: Additions, deletions, and changes to monitor resources
+- Notebooks: Additions, deletions, and changes to notebook resources
+- Request: All web requests
+- Security Monitoring: Additions, deletions, and changes to security monitoring resources
+
+{{< img src="account_management/audit_logs/event-types.png" alt="Audit Logs event types setup in Datadog" style="width:70%;">}}
+
+### Archiving
+
+Archiving is an optional feature for Audit Logs. You can use archiving to write to Amazon S3, Google Cloud Storage, or Azure Storage and have your SIEM system read events from it. Archiving is a double-write feature that has variable write times, but on average all events are written within 15 minutes of generation.
+
+To enable archiving for Audit Logs, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*. Scroll down to Archiving and click the Store logs toggle to enable.
+
+### Retention
+
+Retaining logs is an optional feature for Audit Logs. To enable, navigate to your [Organization Settings][2] and select *Audit Logs Settings* under *Security*.  Scroll down to Retention and click the Retain logs toggle to enable.
+
+The default retention period for an audit log is 7 days. You can set a retention period between 3 and 90 days.
+
+{{< img src="account_management/audit_logs/retention.png" alt="Audit Logs Retention setup in Datadog" style="width:80%;">}}
+
+Note: Audit Logs are priced as retained logs, and there is not a cost for ingestion or archiving. See the [Log Management pricing page][3] for more information.
+
+## Explore Audit Logs
+
+To explore an audit log, navigate to the [Audit Logs][1] section, also accessible from your [Organizational Settings][2] in the Datadog app.
+
+{{< img src="account_management/audit_logs/explore-audit-logs.png" alt="Audit Logs in the Organization Settings menu" style="width:50%;">}}
+
+Audit Logs have the same functionality as logs within the Datadog [Logs Explorer][4]:
+
+- Filter by facets to drill down into audit logs by `Status` (`Error`, `Warn`, `Inf`o) or `Method` (`POST`, `GET`, `DELETE`).
+- For compliance audits, filter by `event_name` and select `authentication only` to see compliance-related events.
+- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.
+
```

Comment:
@sidph I wonder if it makes sense to be clear about which fields are not expected to be stable enough to write monitors / scripts on. Particularly the description and the prev/new value facets I'd call out as for human eyes. Thoughts?

Reply:
+1, definitely would be helpful to make note of any fields that won't be stable for beta.

Reply:
I'd reword this one to - 
- Filter to drill down into audit logs by Event Names (Dashboards, Monitors, Authentication, etc), Authentication Attributes (Actor, API Key ID, User email, etc), `Status` (`Error`, `Warn`, `Info`), `Method` (`POST`, `GET`, `DELETE`), and other facets
- Drill down into related audit logs by selecting a log and navigating to the event attributes tab. Select a specific attribute to filter by or exclude from your search, such as `http.method`, `usr.email`, `client.ip`, etc.